dom/calculator.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer src="exercise6-calculator.js"></script>
+    <link rel="stylesheet" href="calculator.css">
+    <title>Calculator Program</title>
+</head>
+<body>
+    <main>
+        <h1 class="title">Calculator Program</h1>
+
+        <div id="calc-grid-container">
+            <div id="display-box"></div>
+            <input type="button" class="button" id="button0" value="0" onclick="displayValue('0')">
+            <input type="button" class="button" id="button1" value="1" onclick="displayValue('1')">
+            <input type="button" class="button" id="button2" value="2" onclick="displayValue('2')">
+            <input type="button" class="button" id="button3" value="3" onclick="displayValue('3')">
+            <input type="button" class="button" id="button4" value="4" onclick="displayValue('4')">
+            <input type="button" class="button" id="button5" value="5" onclick="displayValue('5')">
+            <input type="button" class="button" id="button6" value="6" onclick="displayValue('6')">
+            <input type="button" class="button" id="button7" value="7" onclick="displayValue('7')">
+            <input type="button" class="button" id="button8" value="8" onclick="displayValue('8')">
+            <input type="button" class="button" id="button9" value="9" onclick="displayValue('9')">
+            <input type="button" class="button" id="button-decimal" value="." onclick="displayValue('.')">
+            <input type="button" class="button" id="button-add" value="+" onclick="displayValue('+')">
+            <input type="button" class="button" id="button-minus" value="-" onclick="displayValue('-')">
+            <input type="button" class="button" id="button-multiply" value="x" onclick="displayValue('*')">
+            <input type="button" class="button" id="button-divide" value="/" onclick="displayValue('/')">
+            <input type="button" class="button" id="button-result" value="=" onclick="doTheMath()">
```

Comment:
remove do math and all that jazz from above

Reply:
removed all functions in html page and onclick elements